Sources/BBCodeParser.php

```diff
@@ -62,63 +62,63 @@ class BBCodeParser
 	 *
 	 * If not empty, only these BBCode tags will be parsed.
 	 */
-	public $parse_tags = [];
+	public array $parse_tags = [];
```

Comment:
All of the type hinting changes in this class are good changes to make, but they probably belong in a separate PR.